src/_utils/focusVisibleProvider/index.unit.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react'
+import { act } from 'react-dom/test-utils'
+import { mount } from 'enzyme'
+
+import FocusVisibleProvider, { FocusVisibleContext } from '.'
+import { KEY } from '_utils/keycodes'
+
+const ChildComponent = () => {
+  const hadKeyboardEvent = useContext(FocusVisibleContext)
+  return <input type="button" className={hadKeyboardEvent.toString()} value="Test" />
```

Comment:
It look like the input is only here to have a `className` prop which is then only used to check the value of the context. maybe we can simplify it like this:

```tsx
describe('FocusVisibleProvider', () => {
  // ...
  let focusVisibleContext = null

  const ChildComponent = () => {
    focusVisibleContext = useContext(FocusVisibleContext)
    return null
  }

  // ...

  afterEach(() => {
    // ...
    focusVisibleContext = null
  })

  it('Should provide "false" for hadKeyboardEvent value on mounting', () => {
    expect(focusVisibleContext).toEqual(false)
  })

  // ...
}
```

What do you think?

Reply:
Yes you right it'll be clearer to only check the context value without using a DOM element.

Reply:
Done